src/provider/provider_cuda.c

```diff
@@ -404,6 +416,99 @@ static const char *cu_memory_provider_get_name(void *provider) {
     return "CUDA";
 }
 
+typedef CUipcMemHandle cu_ipc_data_t;
```

Comment:
please keep this close to other typedefs, it would be easier to search for it

Reply:
Done